external/js/cothority/src/network/connection.ts

```diff
@@ -104,7 +110,7 @@ export class WebSocketConnection implements IConnection {
 
     /** @inheritdoc */
     getURL(): string {
-        return this.url.origin;
+        return this.url.href;
```

Comment:
It's breaking retrocompat; but if it's wanted, we can also directly return a copy of `this.url` :P

Reply:
gosh, I don't wanna break anything, just restoring the same behaviour as before #2196 ... 😰

Reply:
Yeah, so to have the old behavior would be to explicitly check for a trailing slash and removing it when existing, in getURL and send IMO.

Reply:
yep, that's what I did 
```
        if (this.url.pathname === "/") {
            return this.url.origin;
        }
        return this.url.href;
```

Reply:
Ho, good then, I didn't refresh the page with the latest commit ;)